daemon/api_snapshots.go

```diff
@@ -173,3 +173,24 @@ func getSnapshotExport(c *Command, r *http.Request, user *auth.UserState) Respon
 
 	return &snapshotExportResponse{SnapshotExport: export}
 }
+
+var snapshotImportCmd = &Command{
+	Path: "/v2/snapshot/import",
```

Comment:
we don't have anything like this so far. I think it either needs to be

* /v2/snapshots/import
*  or just /v2/snapshots  , in the latter case it would do something like /v2/snaps and use the Content-Type to know if it's an action (json) or an import

Reply:
I changed the code now to use /v2/snapshots and the content type to differentiate between change and import